### PR TITLE
Add cocoapods kokoro presubmit job.

### PR DESCRIPTION
**Pull request by jverkoey**

### CODE REVIEW

**Comment by jverkoey** [comment]
Both builds are green. Reviewers please take a look.

File: .kokoro [hunk 1]
```diff
@@ -49,6 +49,30 @@ fix_bazel_imports() {
...
+run_cocoapods() {
+  gem install xcpretty cocoapods --no-rdoc --no-ri --no-document --quiet
+  pod --version
+
+  if [ -n "$KOKORO_BUILD_NUMBER" ]; then
+    # Move into our cloned repo
+    cd github/repo
+  fi
+
+  export DEVELOPER_DIR=/Applications/Xcode_8.2.app/Contents/Developer
```
Comment (randallli) [inline .kokoro]: We should have a comment explaining that this is older than our supported Xcode and that we plan on changing it when 8.3.3 becomes available

**Review by randallli — APPROVED** [review]

File: .kokoro (hunk 1 above)
Comment (romoore) [reply .kokoro]: Should I rename my Xcode installations to follow this pattern?  This is mine:

```bash
 ls -1 /Applications/ | grep Xcode
Xcode.app
Xcode8.3.3.app
```

Comment (romoore) [reply .kokoro]: Maybe as a future improvement, just loop over all installed Xcode versions?

```bash
$ mdfind "kMDItemCFBundleIdentifier == 'com.apple.dt.Xcode'"
/Applications/Xcode.app
/Applications/Xcode8.3.3.app
```

Comment (jverkoey) [reply .kokoro]: Done.

**Comment by jverkoey** [comment]
Build breakage is resolved in https://github.com/material-components/material-components-ios/pull/2449.

**Pull request merged**